txnprovider/txpool/pool_txn_parser.go

```diff
@@ -690,13 +690,14 @@ func parseTransactionBodyAA(payload []byte, p int, slot *TxnSlot) (int, error) {
 	if err != nil {
 		return 0, err
 	}
-	slot.SenderAddress = &address
+	slot.SenderAddress = address
+	copy(sender, address[:])
```

Comment:
```suggestion
        if ctx.withSender {
	    copy(sender, address[:])
	}
```

We can also probably set this to paymaster if it isn't `nil` below.